synapse/lib/snap.py

```diff
@@ -414,7 +414,8 @@ async def set(self, name, valu, norminfo=None, ignore_ro=False):
         if propsubs is not None:
             for subname, subvalu in propsubs.items():
                 full = f'{prop.name}:{subname}'
-                if self.form.props.get(full) is not None:
+                subprop = self.form.props.get(full)
+                if subprop is not None and not subprop.locked:
```

Comment:
Worth adding a changelog entry to note this behavior or is it internal enough to skip it?

Reply:
I'll add it as a bug fix :+1: